azurerm/resource_arm_sql_server.go

```diff
@@ -173,6 +206,10 @@ func resourceArmSqlServerRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("location", azure.NormalizeLocation(*location))
 	}
 
+	if identity := resp.Identity; identity != nil {
+		d.Set("identity", flattenAzureRmSqlServerIdentity(*identity))
+	}
```

Comment:
we need to ensure the `identity` block is always set - as such can we make this:

```suggestion
	if err := d.Set("identity", flattenAzureRmSqlServerIdentity(resp.Identity); err != nil {
		return fmt.Errorf("Error setting `identity`: %+v", err)
	}
```

Reply:
Done.